### DISCUSSION TITLE
How to display all Go interfaces that a given type implements in vim mode

### DISCUSSION THREAD

**Discussion by milosgajdos** [zed-industries/zed#16902] [Feature Requests]
I use `vim` with the wonderful [vim-go](https://github.com/fatih/vim-go) and I have this shortcut (irrelevant for the question but posting it for brevity)

```
au FileType go nmap <Leader>f <Plug>(go-implements)
```

What it does is, if I position my cursor on a type it spits out a list of interfaces the given type implements. This is extremely handy to me.

In Zed I noticed the built-in LSP provides the "reverse" of it:

```
      "g i": "editor::GoToImplementation",
```

it seems to find all the types that implement the specific Go interface....is there a way to do what I have in vim? I know vim-go uses [guru](https://github.com/coder/go-tools/tree/master/cmd/guru) for that, so I'm wondering maybe there is a way to integrate it with Zed task runner somehow.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
